.travis.yml

```diff
@@ -8,6 +8,10 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.6-dev"
+  - "3.7-dev"
```

Comment:
Also from the article:

> Since Python 3.7 is under active development I can't guarantee it's stability or that things won't change from underneath you, so I'm not going to recommend testing against it yet.